src/Board.cpp

```diff
@@ -14,4 +17,29 @@ Board::Board(int rowNumber,
     board_.assign(rowNumber_, singleRow);
 }
 
+Board::Board(std::string path)
```

Comment:
Is this constructor needed for now?
We can use the above one like this:
```
    JSONFileReader fileReader(path);
    Board board(fileReader.readRowsNumber(),
                fileReader.readColsNumber(),
                fileReader.readRows(),
                fileReader.readCols());
```

The other option is to take a fileReader as a param:
```
    JSONFileReader fileReader(path);
    Board board(fileReader);
```
```
    Board::Board(JSONFileReader jfr)
        : rowNumber_(jfr.readRowsNumber())
        , colNumber_(jfr.readColsNumber())
        , rows_(jfr.readRows())
        , cols_(jfr.readCols())
    {}
```

and thanks to that fields can be const

Reply:
You said previously "What do you think about a new constructor of the Board class..." ;D

Reply:
Yep :) I had another idea earlier, but the code evolved in another way so now I claim that it's better not to have it :D

Reply:
BAU - Business As Usual - that's a normal programmers work - do one thing then remove it, do another and remove as well, etc. :D